src/components/SubmitForm/SubmitForm.js

```diff
@@ -1,101 +1,131 @@
-                />
-              </>
-            )}
-          />
-        </FieldWrapper>
-        <Button as="button" type="submit" disabled={pristine || invalid || submitting}>
-          Submit
-        </Button>
-      </form>
-    )}
-  />
-)
+    return (
+      <Form
+        onSubmit={({ category, level, text }) => (
+          // firebase.createEntry({
+          //   category: category.value,
+          //   level: level.value,
+          //   text,
+          // })
+          new Promise((resolve, reject) => {
+            console.log(this.mde.value())
+            resolve()
+          })
+        )}
+        render={({ handleSubmit, pristine, invalid, submitting, reset }) => (
+          <form onSubmit={(e) => e.preventDefault() || handleSubmit().then(reset)} noValidate>
+            <FieldWrapper>
+              <Label>Category</Label>
+              <Field
+                name="category"
+                validate={required}
+                render={({ input, meta }) => (
+                  <Select
+                    {...input}
+                    placeholder="Choose category..."
+                    error={meta.error && meta.touched}
+                    options={[
+                      { value: "JavaScript", label: "JavaScript" },
+                      { value: "Python", label: "Python" },
+                      { value: "C++", label: "C++" }
+                    ]}
+                  />
+                )}
+              />
+            </FieldWrapper>
+    
+            <FieldWrapper>
+              <Label>Level</Label>
+              <Field
+                name="level"
+                validate={required}
+                render={({ input, meta }) => (
+                  <Select 
+                    {...input}
+                    placeholder="Choose level..."
+                    error={meta.error && meta.touched}
+                    options={[
+                      { value: "junior", label: "Junior" },
+                      { value: "regular", label: "Regular" },
+                      { value: "senior", label: "Senior" }
+                    ]}
+                  />
+                )}
+              />
+            </FieldWrapper>
+    
+            {/* <FieldWrapper>
+              <Field
+                name="captcha"
+                render={({ input }) => (
+                  <ReCAPTCHA sitekey={process.env.CAPTCHA_PUBLIC} {...input} />
+                )}
+              />
+            </FieldWrapper> */}
+    
+            <FieldWrapper>
+              <Label>Your question</Label>
+              <Field
+                name="mde"
+                validate={() => this.state.mde ? null : "Wymagane"}
+                render={({ meta }) => (
+                  <MDEWrapper error={this.state.mdeTouched && meta.error}>
+                    <SimpleMDE
+                      onFocus={() => this.setState({ mdeTouched: true })}
+                      onChange={value => this.setState({ mde: value })}
```

Comment:
```suggestion
                      onChange={value => {
                        this.setState({ mde: value });
                        onChange(value);
                      }}
```

to powinno załatwić Twój problem z walidacją